toolbox

```diff
@@ -405,6 +412,26 @@ image_reference_has_domain()
 )
 
 
+images_get_details()
+(
+    images="$1"
+
+    if ! echo "$images" | while read -r image; do
+            if ! $prefix_sudo podman images \
```

Comment:
We have to be careful about `image` being an empty string. Fixed in commit 4f2eeb091cdcadf9e4a4d255d6d8bfe649ccd698